ml_tools/cptvfileprocessor.py

```diff
@@ -58,6 +58,8 @@ def needs_processing(self, filename):
 
     def process_folder(self, folder_path, worker_pool_args=None, **kwargs):
         """Processes all files within a folder."""
+        if not os.path.exists(folder_path):
```

Comment:
Should this raise an error or is returning without doing anything ok?

Reply:
added an error log. I was getting an error here when I think because i was running train at the same time. probably shouldn't occur often.